p4include/pna.p4

```diff
@@ -474,6 +474,50 @@ extern Digest<T> {
 }
 // END:Digest_extern
 
+// BEGIN:MatchValueLookupTable
+extern MatchValueLookupTable<K, V, E> {
```

Comment:
In DPDK, both P4 input and BFRT/P4info Json uses the same name i.e. MatchValueLookupTable?
(In MEV PNA compiler, in P4 input, it is referred as ExactMatchValueLookupTable)

Reply:
Here is what Ruchit Said
[Tuesday 5:28 PM] Kumar, Kamlesh1
Generally we have same typename in runtime files as in p4 program. I feel little weird about ExactMatchValueLookupTable .

[Tuesday 5:29 PM] Gupta, Ruchit
yes I think so, we had zeroed down on the name MatchValueLookupTable and dropped 'exact' from it

Reply:
Ok. Yeah, it is different requirement we got.